docs/explanation/caching.md

```diff
@@ -38,7 +38,80 @@ _Removing the Remote Cache_:
 If users wish to remove the remote cache without deleting the associated AWS S3 bucket, they can use the following command:
 
 ```
-k8sgpt cache remove --bucket <name>
+k8sgpt cache remove
 ```
 
 This command ensures that the cache items are removed from the K8sGPT CLI, but the bucket and its contents in AWS S3 will remain intact for potential future usage.
+
+
+## Azure Blob Storage Integration
+K8sGPT offers integration with **Azure Blob Storage**, a scalable and cost-effective object storage solution provided by **Microsoft Azure**.
+
+### Prerequisites
+Before using the remote caching feature with Azure Blob Storage in K8sGPT, ensure you have the following prerequisites set up:
+
+- `AZURE_STORAGE_ACCOUNT`: An Azure storage account is required to authenticate with azure CLI programmatically.
+
+_Adding a Remote Cache_:
+
+Users can add a remote cache to the K8sGPT CLI by executing the following command:
+
+```
+k8sgpt cache add --azure --storage <azure storage account name> --container <container name>
+```
+
+This command will create a new container in the specified Azure Storage Account if it does not already exist. The created container will serve as the remote cache for K8sGPT.
+
+_Listing Cache Items_:
+
+To view the items stored in the remote cache, users can use the following command:
+
+```g
+k8sgpt cache list
+```
+
+_Removing the Remote Cache_:
+
+If users wish to remove the remote cache without deleting the associated Azure Blob Storage container, they can use the following command:
+
+```
+k8sgpt cache remove
```

Comment:
Can you add what the user is supposed to see e.g. is there a confirmation message?